docs/dev/guidelines/client-tests.rst

```diff
@@ -314,3 +312,7 @@ Some guidelines:
   +--------------------------------------------------------+-----------------------------------------------------------------+
   | A spy should have been called once                     | :code:`expect(spy).toHaveBeenCalledTimes(1);`                   |
   +--------------------------------------------------------+-----------------------------------------------------------------+
+  | A spy should have been called with a value             | Always test the number of calls as well:                        |
+  |                                                        | :code:`expect(spy).toHaveBeenCalledTimes(1);`                   |
+  |                                                        | :code:`expect(spy).toHaveBeenCalledWith(value);`                |
```

Comment:
Would you be so kind to also display how to do this for an nth call?

Reply:
Did it (see new changes); lmk if you would add sth different than I did